### DISCUSSION TITLE
Cross-section report using Latex

### DISCUSSION THREAD

**Discussion by jessebluestein** [robbievanleeuwen/section-properties#212] [Ideas]
I noticed that the cross-section report is something flagged for future development. Do you have any ideas on how you would like to generate the report? I have some experience generating PDF reports using Latex and python and I was wondering if that would be helpful for this

**Comment by robbievanleeuwen** [comment]
Hi @jessebluestein, yes that was a pipe dream that I haven't gotten around to implementing! I've had a quick look at the reports you've generated in your repo ``steel_beam_designer`` and this is pretty much what I had in mind, they look great!

If you've got some time I'd be very open and happy for you to put something together for a pull request, also happy to contribute and assist with any design decisions 😄

**Comment by jessebluestein** [comment]
Sounds good, thanks Robbie! Will do
